src/ng/animate.js

```diff
@@ -180,193 +327,174 @@ var $AnimateProvider = ['$provide', function($provide) {
        * @ngdoc method
        * @name $animate#enter
        * @kind function
-       * @description Inserts the element into the DOM either after the `after` element or
-       * as the first child within the `parent` element. When the function is called a promise
-       * is returned that will be resolved at a later time.
+       * @description Inserts the element into the DOM either after the `after` element (if provided) or
+       *   as the first child within the `parent` element and then triggers an animation.
+       *   A promise is returned that that will be resolved during the next digest once the animation
+       *   has completed.
+       *
        * @param {DOMElement} element the element which will be inserted into the DOM
        * @param {DOMElement} parent the parent element which will append the element as
        *   a child (if the after element is not present)
-       * @param {DOMElement} after the sibling element which will append the element
+       * @param {DOMElement=} after the sibling element which will append the element
```

Comment:
It sounds kind of weird to say that the `after` element will append the element after itself.
I would rather say _"[...] after which the element will be appended"_.